cds-snc/covid-alert-portal
# Remove Registrant Name step

# Summary | Résumé

Removes the Registrant Name step, including:
- Migration to delete the column
- Model change
- Remove URL
- Remove template
- Remove or update related tests
- Change redirect from email_confirm
- Render Flash message (email confirmed) on Address (new next step)
- Update Locale files

This is part of a bigger activity captured in [this trello card](https://trello.com/c/XITLYg8Y/696-update-flow-and-add-pagination-for-the-registration-website).

## To test
- Make sure to run the pending migration: `python manage.py migrate`
- Enter your email/confirm email
- The next step after email confirmation should be "Address" (previously Name)
